modules/networking/main.tf

```diff
@@ -3,6 +3,10 @@ resource "google_compute_network" "default" {
   name                    = "${var.namespace}-vpc"
   description             = "${var.namespace} VPC Network"
   auto_create_subnetworks = false
+  project = var.project
```

Comment:
it seems like different linting is being used? is there pre-commit for this repo?

Reply:
There is, @moonshot-s please make sure you have configured your local development environment: https://github.com/wandb/terraform-google-dagster#development